apiserver/facades/controller/migrationtarget/migrationtarget.go

```diff
@@ -251,7 +247,7 @@ func (api *API) AdoptResources(args params.AdoptResourcesArgs) error {
 		return errors.Trace(err)
 	}
 
-	return errors.Trace(ra.AdoptResources(api.callContext, st.ControllerUUID(), args.SourceControllerVersion))
+	return errors.Trace(ra.AdoptResources(state.CallContext(m.State()), st.ControllerUUID(), args.SourceControllerVersion))
```

Comment:
st

Reply:
No, st here is a PooledState. I needed State.